Sources/TokamakCore/Views/Selectors/Slider.swift

```diff
@@ -0,0 +1,169 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+public enum _SliderStep {
+  case any
+  case discrete(Double.Stride)
+}
+
+func convert<T>(_ binding: Binding<T>) -> Binding<Double> where T: BinaryFloatingPoint {
```

Comment:
I think these `convert` functions should either be static methods on `Slider`, or changed to `fileprivate`.